tests/vlan/test_host_vlan.py

```diff
@@ -98,6 +99,20 @@ def setup_host_vlan_intf_mac(duthosts, rand_one_dut_hostname, testbed_params, ve
     yield
 
     duthost.shell('redis-cli -n 4 hmset "VLAN|%s" mac %s' % (vlan_intf["attachto"], dut_vlan_mac))
+
+    del_vlan_json = json.loads("""
+            {
+                "VLAN":{
+                    "%s":{
+                        "mac": "%s"
+                    }
+                }
+            }
+        """ % (vlan_intf["attachto"], dut_vlan_mac))
+    duthost.copy(content="[" + json.dumps(del_vlan_json, indent=4) + "]", dest='/tmp/del_vlan_mac.json')
```

Comment:
Why run `json.loads` and then `json.dumps`? Can simplify it as below:
```
del_vlan_json = [
            {
                "VLAN":{
                    "%s":{
                        "mac": "%s"
                    }
                }
            }
]
duthost.copy(content=json.dumps(del_vlan_json, indent=4), dest='/tmp/del_vlan_mac.json')
```

BTW, I have concern with `%s` in the del_vlan configlet. It is a placeholder for string formatting. The previous `redis-cli` command didn't really set a VLAN with name `%s`. This char was replaced by actual VLAN name.